### DISCUSSION TITLE
多节点可以共享一个钱包地址吗？

### DISCUSSION THREAD

**Discussion by baysmall1** [filecoin-project/community-china#89] [Bug讨论]
当我部署多个节点，每个节点都将产生自己的钱包启动后的地址，这样管理不太方便。大家有什么好的建议吗？

**Comment by waynewyang** [comment]
可以的。

**Reply by baysmall1** [reply]
请问具体应该怎么操作 ？ 把/var/lib/bee/下的文件都复制到新节点么？

**Reply by waynewyang** [reply]
你可以采用lotus-wallet，作为钱包服务器
其余的节点都从lotus-wallet 去获取

具体的操作
1 你的多个node全节点config.toml配置remote_end指向你的lotus_wallet服务器
2 你的lotus-wallet上存放私钥
但是需要做好安全工作，防止私钥的暴露。

**Reply by waynewyang** [reply]
make lotus-wallet编译钱包服务

代码[lotus-wallet](https://github.com/filecoin-project/lotus/tree/master/cmd/lotus-wallet)

**Reply by beck-8** [reply]
> 请问具体应该怎么操作 ？ 把/var/lib/bee/下的文件都复制到新节点么？

bee？你走错地方了吧？

**Reply by Osub** [reply]
> 请问具体应该怎么操作 ？ 把/var/lib/bee/下的文件都复制到新节点么？

/etc/bee/bee.yaml  中   password-file: 配置密码路径 
😁
